### DISCUSSION TITLE
Hide MacOS top bar navigation menu, while using Launch from Browser

### DISCUSSION THREAD

**Discussion by ManticoreM1A2** [SafeExamBrowser/seb-mac#442] [Q&A]
Greetings All,

there is a problem with my safe exam browser, i am using moodle
the thing is that when i launch the quiz from the Launch menu like in the figure 1,
<img width="1155" alt="Screenshot 2024-09-30 at 9 50 13 AM" src="https://github.com/user-attachments/assets/c60a2280-c42b-4459-a365-6fe310e9a3c4"> figure 1

the top task bar on the Mac is still present and does not disappear like in the figure 2,
![IMG20240930093916](https://github.com/user-attachments/assets/95d04432-593c-4176-a81a-d07aafdcd6ca)
figure 2

is there any solution for that?, since i have noticed that the only way to make the task bar disappear is if i have launched it by downloading the config first and than opening the config file 

BR,

**Comment by danschlet** [comment]
What you are referring to is called the Menu Bar.

There is a SEB setting to disable the menu bar ([see manual](https://safeexambrowser.org/macosx/mac_usermanual_en.html#AppearancePane)), but you would need to create and upload your own config file to the Moodle quiz. 

Many people want to have the menu bar visible, because students can use it to reconnect to a Wi-Fi network if connection is lost. SEB in general blocks third-party menu bar items to be invoked. Did you encounter a security issue when the menu bar is displayed?
